### DISCUSSION TITLE
Update Angular plugin

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Knowlg/Community#97] [General Discussions]
As Angular Migration for SunbirdEd . Team we are removing some of the dependency please remove the dependency of the same in your repo.
1. angular2-uuid
2. ng2-cache-service
3.ngx-infinite-scroll

please refer the ED ticket [ED-1277](https://project-sunbird.atlassian.net/browse/ED-1277)
cc: @Krishnaj20 @vinukumar-vs

**Comment by HarishGangula** [comment]
@rajeevsathish  We have used the above packages in collection editor v2, we will analyze and plan for removal of the these dependencies 

@pallakartheekreddy @Krishnaj20

**Reply by vinukumar-vs** [reply]
Thanks for the confirmation.. 
We are running with very tight timeline to roll-out the Angular upgrade by March, 2023 end. 

Can we know in which release are you planning this & when it will be available.

@Krishnaj20 @pallakartheekreddy  @rajeevsathish

**Reply by pallakartheekreddy** [reply]
@vinukumar-vs We are removing the above dependencies in the following release
**Sunbird-Knowlg** - Release-5.4.0 
**Sunbird-inQuiry** - Release-5.5.0
